conf/modules.config

```diff
@@ -903,6 +903,37 @@ process{
         ext.prefix       = {"${meta.id}.somatic_snvs"}
     }
 
+    // VCF QC
+    withName: 'BCFTOOLS_STATS'{
+        ext.when         = { !(params.skip_tools && params.skip_tools.contains('bcftools')) }
+        ext.prefix       = { "${meta.variantcaller}_${vcf.baseName.minus(".vcf")}" }
```

Comment:
Can't we remap id based on meta.variantcaller + meta.sample

Reply:
that way, we just keep it simple here

Reply:
problem is, tools like strelka that have multiple vcfs for one sample; like `sample2.genomes.vcf.gz` and `sample2.variants.vcf.gz`

Reply:
https://github.com/nf-core/sarek/blob/68b9930a74962f3c42eee71f51e6dd2646269199/main.nf#L3429 looking here it would also effect Manta & strelka somatic, and now also deepvariant